source/documentation/data-docs/curated-databases-docs/databases-for-analysis-and-apps.md

```diff
@@ -0,0 +1,58 @@
+
+#### Data Minimisation
+
+When writing data to your app-bucket you should only ever give the app what it needs. I.e. even though your analysis might require personal or low level data. If the calculations on your App do not need the data at this lower level, then only write data to the app-bucket that is an aggregate of the data you take. You should never put data in the app-bucket that you do not want users to see (even if the code behind the app itself has no method to expose that data to the user via the app).
+
+
+#### Writing data to an App bucket
+
+There are multiple ways you can do this and it is up to you to determine how best to do it.
+
+- You could manually run a script from your AP User instance that writes the necessary data to the app bucket whenever it needs updating
+- You could write an airflow script that writes data from a database to the app bucket (this is because the airflow role will be seperate to the app role). Meaning the app still only has access to it's very specific area in AWS.
+- If you want a database backend but nothing as big as Athena you could write your data into an SQLite and save to to the app-bucket. Then just like how R/Python would read in a file from S3. You could load in SQLite database into memory and use that as a backend database for your app.
+
+> When updating data for Rshiny Apps you often need to restart the R session so it refreshes the cached data. You can do this using airflow (to see how to do this search the airflow-dags repository for the `BashOperator`).
+
+
+#### Creating a database for your app
+
+> Note this is only recommended for users who understand how to create/manage databases on Athena. See other parts of guidance on these tools for more details on how to use them.
+
+In most cases you will not need to do this and it is unnecessary. However, if you feel like you do need to create an Athena SQL database because your app requires the processing power from Athena and the data you are reading in is too large to be cached and there is no way around it then you will need to do the following:
+
+- Create an athena database that will be used for your app (the data will still sit in the apps S3 bucket but it will also need a Glue schema created in order for Athena to query it).
+- Message on the data-engineer slack channel to ask them to add this database for your app.
+
+Your app role will then be given access to only read the specific database for your app and no other databases. This again ensures the airgap between what an app can access and the wider resources given to AP-users. If you require to update the app database follow the same tips / routes to writing data in S3 in the ["writing data to an app bucket" section](#writing-data-to-an-app-bucket).
+
+If exposing your App to SQL you need to be careful with SQL injection. You should avoid things like f-strings or inserting values into SQL strings. Or use a tool as your sql interface to deal with that like sqlalchemy.
```

Comment:
Your last two sentences ought to be one sentence with a comma instead of the full stop, and you should put SQL not sql, and you should link to sqlalchemy if you're mentioning it, and I shouldn't do run-on sentences with loads of 'and' clauses because it's hard to read and keep track of.